roles/cockpit/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+#########################################################################
+# Title:         Sandbox: Cockpit Role                                  #
+# Author(s):     CHAIR/Raneydazed                                       #
+# URL:           https://github.com/saltyorg/Sandbox                    #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Add DNS record
+  ansible.builtin.include_tasks: "{{ resources_tasks_path }}/dns/tasker.yml"
+  vars:
+    dns_record: "{{ lookup('vars', role_name + '_dns_record') }}"
+    dns_zone: "{{ lookup('vars', role_name + '_dns_zone') }}"
+    dns_proxy: "{{ lookup('vars', role_name + '_dns_proxy') }}"
+  when: not cockpit_traefik_enabled
```

Comment:
Shouldn't this be reversed? We want to set the DNS records when Traefik will be enabled

Reply:
Shit you're right. I had my head wrapped around this the wrong way. 😂